Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -142,6 +147,10 @@ void MemoryViewWidget::Update()
         if (PowerPC::HostIsRAMAddress(address))
         {
           hex_item->setText(value_to_string(address));
+          if (columns == 2 && c == 0)  // 2 Column mode: Report hex value in first column.
```

Comment:
Comments should go above the line when possible. This conditional should also be braced since the body travels more than one line.